purchase_order_revision/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013 Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+# Copyright 2015 Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+# Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+# Copyright 2018 Ramon Bajona <ramon.bajona@guadaltech.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase order revisions",
+    "version": "11.0.1.0.0",
+    "category": "Purchase Management",
+    "author": "Agile Business Group,"
+              "Camptocamp,"
```

Comment:
Add space after comma for better text display
![image](https://user-images.githubusercontent.com/5856745/63469006-313f7b00-c437-11e9-9438-bf6acb21df33.png)